script/linux

```diff
@@ -12,10 +12,9 @@ function finalize {
   # after packages install (curl, etc), get the rust toolchain
   which rustup > /dev/null 2>&1 || curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
   # verify the mold situation
-  if ! command -v mold >/dev/null 2>&1; then
-    echo "Warning: Mold binaries are unavailable on your system." >&2
-    echo "    Builds will be slower without mold. Try: script/install-mold" >&2
-  fi
+  echo "Note: It's recommended to install and configure Wild or Mold for faster builds."
+  echo "      Run script/install-wild or script/install-mold."
+  echo "      Then see doc/src/development/linux.md to set your cargo config."
```

Comment:
Could we link/be a bit more specific about where to look in linux.md?

Alternatively (if you feel comfortable with it) we can do the cargo config change from the script. (lets also echo what we are doing and how to undo it)

Reply:
> I think changing the user wide config feels a bit intrusive for users who work on projects other than Zed.

Good point. Can we print the instructions at the end of the script?